dashboard/config/routes.rb

```diff
@@ -213,6 +213,8 @@ module OPS
   post '/admin/confirm_email', to: 'admin_users#confirm_email', as: 'confirm_email'
   post '/admin/undelete_user', to: 'admin_users#undelete_user', as: 'undelete_user'
 
+  get '/admin/styleguide', :to => 'admin_styleguide#show', as: 'styleguide'
```

Comment:
Hm, good Q: re where to put it. Brainstorming a bit:

* `studio.code.org/styleguide`
* `studio.code.org/dev/styleguide`
* `studio.code.org/test/styleguide`
* `studio.code.org/about/styleguide`